lib/attr_encrypted/adapters/active_record.rb

```diff
@@ -16,10 +16,9 @@ def reload_with_attr_encrypted(*args, &block)
             alias_method_chain :reload, :attr_encrypted
 
             attr_encrypted_options[:encode] = true
+
             class << self
-              alias_method :attr_encryptor, :attr_encrypted
```

Comment:
Oh ok nice catch, missed that this was only removed in `adapter/active_record.rb`, makes sense to me :+1: